marconi-chain-index/src/Marconi/ChainIndex/Indexers/MintBurn.hs

```diff
@@ -477,35 +492,66 @@ instance RI.Queryable MintBurnHandle where
       mkRowPredicates = \case
         QueryAllMintBurn maybeSlotNo -> upToSlot maybeSlotNo
         QueryAllBurn maybeSlotNo -> upToSlot maybeSlotNo <> [isBurn]
-        QueryByAssetId policyId assetName maybeSlotNo -> upToSlot maybeSlotNo <> [matchesAssetId policyId assetName]
-        QueryBurnByAssetId policyId assetName maybeSlotNo -> upToSlot maybeSlotNo <> [matchesAssetId policyId assetName, isBurn]
+        QueryByAssetId policyId assetName maybeSlotNo ->
+          upToSlot maybeSlotNo <> [matchesPolicyId policyId] <> [matchesAssetName assetName]
+        QueryBurnByAssetId policyId maybeAssetName maybeSlotNo maybeTxId ->
+          [matchesPolicyId policyId, isBurn]
+            <> [matchesAssetName maybeAssetName, isBurn]
```

Comment:
`isBurn` is checked twice?

Reply:
changed this, it should only be once, a cop/paste error on my part